frappe/translations/fr.csv

```diff
@@ -246,7 +246,7 @@ Start Import,Démarrer l'import,
 State,Etat,
 Stopped,Arrêté,
 Subject,Sujet,
-Submit,Soumettre,
+Submit,Valider,
```

Comment:
@FHenry 
<img width="682" alt="image" src="https://user-images.githubusercontent.com/28212972/160792201-c7555cc2-c0f8-4d8d-8cdc-aafc15d054ec.png">

<img width="686" alt="image" src="https://user-images.githubusercontent.com/28212972/160792615-7533fa0f-d901-41ae-bf41-839788a9fe93.png">


any specific reason for the change?

Reply:
@hasnain2808 Hi, in fact "Soumettre" can have a "kinky" meaning and it never use in other french applications. In French we do not say "submit a form" (it's a technical wording understand only by developpers)  we prefer to say "validate a form".
The way it is use in Frappe is really when you want to change document status from Draft to => "next status", This is commonly call Validate the document in French.

Reply:
Sample from Dolibarr ERP/CRM
![image](https://user-images.githubusercontent.com/1050053/160818440-6cbb24cb-cceb-47db-982b-c280196825db.png)

Reply:
Sample from Odoo
![image](https://user-images.githubusercontent.com/1050053/160821373-2c178a8a-c1c8-4156-bf92-a37c0f0daf4b.png)